Lib/tkinter/__init__.py

```diff
@@ -847,7 +847,7 @@ def tk_focusNext(self):
         The focus order first goes to the next child, then to
         the children of the child recursively and then to the
         next sibling which is higher in the stacking order.  A
-        widget is omitted if it has the takefocus resource set
+        widget is omitted if it has the takefocus option set
```

Comment:
Formatting, or crossref if possible:

```suggestion
        widget is omitted if it has the *takefocus* option set
```

Reply:
Posted this before seeing the discussion about omitting formatting from docstrings. This should be consistent with whatever is decided.